commands/azureCommands/run-buildTask.ts

```diff
@@ -1,16 +1,16 @@
-import { BuildTaskBuildRequest } from "azure-arm-containerregistry/lib/models";
```

Comment:
You may change the file name from `run-buildTask` to `run-Task`

Reply:
I will make all the refactoring within another PR. In here are only functional changes, and minor refactoring changes.